packit/upstream.py

```diff
@@ -928,7 +928,15 @@ def _fix_specfile_to_use_local_archive(
             "PACKIT_RPMSPEC_RELEASE": self.upstream.get_spec_release(),
             "PACKIT_PROJECT_COMMIT": current_commit,
             "PACKIT_PROJECT_ARCHIVE": archive,
+            "PACKIT_PROJECT_BRANCH": sanitize_branch_name_for_rpm(
+                self.upstream.local_project.ref
+            ),
         }
+
+        # in case we are given template as a release suffix
+        if release_suffix:
+            release_suffix = release_suffix.format(**env)
```

Comment:
That's a logical usecase. We just need to document this properly.

Reply:
that's why it's not marked in the template as done :) :D

Reply:
packit/packit.dev#442